### ISSUE TITLE
Fix Inertia Head type

### ISSUE THREAD

**Issue by ozziexsh** [ozziexsh/laravel-jetstream-react#1]
We currently need to import the `Head` component from inertia like this

This will allow us to remove the 

```typescript
// @ts-ignore
import { Head } from '@inertia/inertia-react';
```

because the type isn't being exported from the types file

https://github.com/inertiajs/inertia/blob/master/packages/inertia-react/index.d.ts#L9

but you can see that the implementation is being exported

https://github.com/inertiajs/inertia/blob/master/packages/inertia-react/src/Head.js

https://github.com/inertiajs/inertia/blob/master/packages/inertia-react/src/index.js#L4

**Comment by ozziexsh** [comment]
looks like an open pr is ready for this here 

https://github.com/inertiajs/inertia/pull/855

and the related issue

https://github.com/inertiajs/inertia/issues/824

**Comment by ozziexsh** [comment]
fixed in e786bb761dd3f04ae7d2c89143cea2d522fc448c
